stdlib/3/tokenize.pyi

```diff
@@ -41,7 +42,11 @@ def detect_encoding(readline: Callable[[], bytes]) -> Tuple[str, Sequence[bytes]
 def tokenize(readline: Callable[[], bytes]) -> Generator[TokenInfo, None, None]: ...
 def generate_tokens(readline: Callable[[], str]) -> Generator[TokenInfo, None, None]: ...
 
-def open(filename: Union[str, bytes, int]) -> TextIO: ...
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    def open(filename: PathLike) -> TextIO: ...
```

Comment:
This still needs to explicitly accept str, bytes, and int (see also the CI failure).

Reply:
Oh, right. Fixed.